File: leetcode1446.cpp
class Solution {
public:
    int maxPower(string s) {
      int cnt = 1,ans = 0;
      for(int i = 1;i<s.length();i++){
        if(s[i-1] == s[i]) cnt ++;
        else{
          ans = max(ans,cnt);
          cnt = 1;
        }
      }
      ans = max(ans,cnt);
      return ans;
    }
};

File: leetcode1446.py
class Solution:
    def maxPower(self, s: str) -> int:
      cnt: int = 1
      ans:int = 0
      for i in range(1,len(s)):
        if s[i - 1] == s[i]:
          cnt += 1
        else:
          ans = max(ans,cnt)
          cnt = 1
      ans = max(ans,cnt)
      return ans
